charts/firefly/values.yaml

```diff
@@ -207,6 +207,9 @@ core:
     serviceMonitor:
       enabled: false
       scrapeInterval: 10s
+    alertRule:
+      enabled: false
+      severity: ""
```

Comment:
Can go ahead and give severities the common default values of `critical` and `warning`